src/main/java/com/carrotcreative/cream/loaders/retry/RetryLoader.java

```diff
@@ -4,14 +4,14 @@
 
 public abstract class RetryLoader {
 
-    protected static final int START_RETRY_SECONDS = 2;
-    protected static final int MAX_RETRY_SECONDS = 64;
-    protected static final float RETRY_SECONDS_GROWTH_RATE = 1.5f;
+    private static final int START_RETRY_SECONDS = 2;
+    private static final int MAX_RETRY_SECONDS = 64;
+    private static final float RETRY_SECONDS_GROWTH_RATE = 1.5f;
 
     protected int mAttemptNumber = 0;
-    protected int mRetrySeconds;
+    private int mRetrySeconds;
```

Comment:
This was set as protected so users can use this if they possibly want to change the logic of the retry method in a subclass